### ISSUE TITLE
can i show mention name without showing @

### ISSUE THREAD

**Issue by khushboojainOffbeat** [armcha/AutoLinkTextView#29]
I need to show mention name without showing @ and also need click event. can you please help me and also wants to know should i set multiple custom AutoLinkMode in single textview?

**Comment by Sayyedrizwan** [comment]
No, if you will not use @ then how program will know what to pick up..
